src/ConnectionTesters/LambdaConnectionTester.ts

```diff
@@ -19,7 +19,8 @@ export class LambdaConnectionTester implements connectionTester {
             InvocationType: "RequestResponse",
             Payload: JSON.stringify({
                             endpointAddress: endpoint.address,
-                            endpointPort: endpoint.port
+                            endpointPort: endpoint.port,
+                            connectionTimeout_ms: timeout_ms
```

Comment:
YAGNI

Reply:
Not sure what you mean - if there is no timeout value passed in then the default timeout is 120 secs (from the OS socket options), which is a long time for a test.